R/params.R

```diff
@@ -175,6 +175,13 @@ params_configurable <- function(param) {
   length(param$value) <= 1 && !is.null(params_get_control(param))
 }
 
+# Returns a new empty named list.
+params_namedList <- function() {
+  empty <- list()
+  names(empty) <- character()
+  empty
+}
```

Comment:
FYI, there is a `setNames()` function, e.g. you can just return `setNames(list(), character())`.